src/lib/utils/meta_tables/abstract_meta_system_table.cpp

```diff
@@ -0,0 +1,55 @@
+#ifdef __linux__
+
+#include <fstream>
+
+#endif

```

Comment:
I'd include fstream in any case. Better to keep the divergence as small as possible.